tss-esapi-sys/.cargo/config

```diff
@@ -0,0 +1,7 @@
+# Cargo seems to expect the -gcc variant instead of -ld
```

Comment:
Is this file needed? I only put it in Parsec because `cargo` was using the system linker (`cc`) and it was failing but for rust-psa-crypto and parsec-se-driver, cross compilation worked fine without.

Reply:
True, I'll check if it works without